Blizzard.IO.Core/IPublisher.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Blizzard.IO.Core
+{
+    public interface IPublisher<in TData>
```

Comment:
Why the in?

Reply:
Rozy told me to use it so you could send data in the inheritance tree

Reply:
Rozi and I talked about it and we did not understood this keyword correctly
we decided not to use it for now

Reply:
f